umap/static/umap/js/modules/form/fields.js

```diff
@@ -212,8 +218,8 @@ Fields.Input = class extends BaseElement {
 }
 
 Fields.BlurInput = class extends Fields.Input {
-  getSyncEvent() {
-    return 'blur'
+  listenForSync() {
+    this.input.addEventListener('blur', () => this.sync())
```

Comment:
No debounce for blur inputs?

Reply:
I don't think it's needed, given for blur input, the sync is only done once (at… "blur" :) ), while for normal input (and textarea) it was done at each character change.